target_redshift/db_sync.py

```diff
@@ -690,13 +691,22 @@ def load_csv(self, s3_key, count, size_bytes, compression=False, manifest=False)
                     insert_sql = """INSERT INTO {} ({})
                         SELECT {}
                         FROM {} s
+                        LEFT JOIN {} target ON {}
```

Comment:
INSERT INTO target

SELECT * FROM stage
LEFT JOIN target ON target.id = stage.id
WHERE target._sys_updated_at IS NULL

Basically only insert if the target doesn't have the record

Reply:
why not make target.id is null as the insert condition?   the above will work if sys updated at is always populated upon initial insert, but comparing against id is good since it is using the unique identifier for the thing we are trying to insert.

Reply:
yea good point I think I just got in a habit of _sys_updated_at stuff